Mogakco/Sources/Domain/UseCases/CreateStudyUseCase.swift

```diff
@@ -22,11 +22,21 @@ struct CreateStudyUseCase: CreateStudyUseCaseProtocol {
         self.userRepository = userRepository
     }
     
-    func create(study: Study) -> Observable<Study> {
+    func create(study: Study) -> Observable<Void> {
         var study = study
         return userRepository.load()
             .compactMap { $0.id }
             .map { study.userIDs.append($0) }
```

Comment:
```suggestion
        return userRepository.load()
            .map { $0.id }
            .map { study.userIDs.append($0) }
```
optional 값이 없어서 map으로 변경되도 될 것 같아요!

Reply:
감사합니다. id는 옵셔널하지 않으니까 그냥 map을 써도 되겠네요